e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-sort.spec.js

```diff
@@ -85,7 +85,6 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
         .contains(ALL_DEALS[19]);
 
       dashboardDeals.next().click();
-      dashboardDeals.exporterButton().click();
```

Comment:
1. Why these button click have been removed - Are they frivolous elements in user journey?
2. Does this PR deals just with exporter column sorting or any column, if any column then we need test cases for most common columns please.

Reply:
1) they were there before as it needed to reclick on 2nd page for sorting again but now it's passed in the session so not needed or breaks the test

2) only sortable column

Reply:
If only sortable column, then do we need to add test cases for those sortable columns please?